ui/pages/Project/components/SavedVariants.jsx

```diff
@@ -115,7 +115,7 @@ const BaseProjectSavedVariants = React.memo(({ project, analysisGroup, loadProje
 
   const getUpdateTagUrl = (newTag) => {
     const isCategory = categoryOptions.includes(newTag)
-    props.updateTable({ categoryFilter: isCategory ? newTag : null })
+    props.updateTableField('categoryFilter')(isCategory ? newTag : null)
```

Comment:
this changed because the new `StateChangeForm` requires a different signature for change functions than before